database/sql/src/main/scala/cromwell/database/sql/MetadataSqlDatabase.scala

```diff
@@ -78,10 +78,11 @@ trait MetadataSqlDatabase extends SqlDatabase {
                           rootWorkflowIdKey: String,
                           labelMetadataKey: String,
                           limit: Int,
+                          nextSummaryPointerUpdateCeiling: Option[Long],
```

Comment:
scaladoc

Reply:
Added. And for your reference: `@param nextSummaryPointerUpdateCeiling If provided, the highest value to which the summary status pointer can be updated. If not set, the pointer will not move.`